liferay-echo/liferay-portal
# LPS-170606 View mode styles for basic components

The goal of this ticket is to decouple basic component styles from Clay. I have basically copied styles from Clay to each individual basic component so they look perfect.

In order to test this:
- Create a page, add all basic components, and check it looks fine.
- Add a remote app (Theme CSS) with a blank stylesheet for Clay and Theme CSS.
- Add the remote app to the created page, check it still looks fine.

Here you have a demo of a page with existing styles (left) and another page with all styles removed (right). The different font is expected, as the added stylesheet is completely empty.

![Peek 2022-12-13 15-21](https://user-images.githubusercontent.com/800645/207358396-6c6b4750-32cf-4c05-bf68-f104a498da7e.gif)
